src/algolia-projects.json

```diff
@@ -1,4 +1,16 @@
 [{
+    "name": "Atom.io",
```

Comment:
I agree, we should write something different. Do you have something in mind? What about `Algolia for Atom`?